bokehjs/src/less/examiner.less

```diff
@@ -137,16 +141,20 @@
   border-width: 1px;
   border-style: solid;
   border-radius: 4px;
-  color: #202124;
-  background-color: #f1f3f4;
-  border-color: #cacdd1;
+  color: var(--color);
+  // @mattpap - I couldn't find a good substitute for this color from the colors
+  // already defined in base.less. We might need something like "muted
+  // background color"
+  background-color: var(--muted-bg-color);
+  border-color: var(--border-color);
```

Comment:
This should use the "default" gray (like of widgets' background). The muted appearance comes from `opacity: 0.6` for non-dirty properties and if dirty it reverts to 1.0.

Reply:
```css
.prop > * {
  opacity: 0.6;
}
.prop.dirty > * {
  opacity: 1.0;
}
```